### PR TITLE
fix(arc-1653): fix sorting filter like design

### PR DESCRIPTION
**Pull request by WardVerc1**
image
Ticket: https://meemoo.atlassian.net/browse/ARC-1653

 Can't seem to find a better solution 👶

### CODE REVIEW

File: src/modules/visitor-space/components/FilterMenu/FilterMenu.module.scss [hunk 1]
```diff
@@ -169,4 +169,17 @@ $checkbox-size: $spacer-sm;
...
+
+	&__sorting-container {
+		margin-top: -1px;
+		padding-left: 1.7rem;
+
+		div {
+			padding: 0;
+
+			div > ul > li > div > div > button {
+				padding-left: 1.6rem !important;
```
Comment (silkederudder-shd) [inline src/modules/visitor-space/components/FilterMenu/FilterMenu.module.scss:181]:
```suggestion
				padding-left: $spacer-sm !important;
```

File: src/modules/visitor-space/components/FilterMenu/FilterMenu.module.scss [hunk 2]
```diff
@@ -169,4 +169,17 @@ $checkbox-size: $spacer-sm;
...
+		padding-left: 1.7rem;
+
+		div {
+			padding: 0;
+
+			div > ul > li > div > div > button {
+				padding-left: $spacer-sm !important;
+			}
+		}
+	}
```
Comment (bertyhell) [inline src/modules/visitor-space/components/FilterMenu/FilterMenu.module.scss:184]: The selector essentially says: after the first dropdown trigger, find a menu that is open and add padding

"+" => elements that follows on the same level in the dom tree
https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator

```suggestion
	// Add padding to the sort search results dropdown
	:global(.c-dropdown__trigger:first-child + .c-dropdown__content-open) {
		:global(.c-menu) {
			padding: 0;
			margin-left: $spacer-sm;
		}
	}
```

**Review by bertyhell — CHANGES_REQUESTED** [review]
see code remark

File: src/modules/visitor-space/components/FilterMenu/FilterMenu.module.scss [hunk 3]
```diff
@@ -169,4 +169,16 @@ $checkbox-size: $spacer-sm;
...
+
+	:global(.c-dropdown__trigger:first-child + .c-dropdown__content-open) {
+		:global(.c-menu) {
+			margin-top: -1px;
+			padding: 0;
+			margin-left: $spacer-sm;
+
+			div > ul > li > div > div > button {
+				padding-left: $spacer-sm !important;
+			}
```
Comment (bertyhell) [inline src/modules/visitor-space/components/FilterMenu/FilterMenu.module.scss:181]: You don't need this anymore, it should already add a margin by setting margin-left on line 177
```suggestion
```

**Review by bertyhell — APPROVED** [review]

**Review by silkederudder-shd — APPROVED** [review]

**Pull request merged**